superset-frontend/src/components/Chart/ChartErrorMessage.tsx

```diff
@@ -21,10 +21,16 @@ import React from 'react';
 import { SupersetError } from '@superset-ui/core';
 import { useChartOwnerNames } from 'src/hooks/apiResources';
 import ErrorMessageWithStackTrace from 'src/components/ErrorMessage/ErrorMessageWithStackTrace';
+import { ChartSource } from 'src/types/ChartSource';
 
-interface Props {
+export interface Props {
   chartId: string;
   error?: SupersetError;
+  subtitle: JSX.Element;
+  copyText: string;
+  link: string | undefined;
```

Comment:
It's equivalent to 
```suggestion
  link?: string;
```

Reply:
There's gotta be a lint rule for that we can add if there's a preference here.